README.md

```diff
@@ -1,46 +1,65 @@
-# Specialist publisher
-
-## Purpose
+# Specialist Publisher Rebuild
 
 Publishing App for Specialist Documents and Manuals.
 
+## Screenshots
+
+![Homepage/Manuals](docs/screen-shot-specialist-publisher-manuals.png)
+*Homepage/Manuals*
+
+![Specialist Documents - CMA Case Format](docs/screen-shot-specialist-publisher-cma-cases.png)
+*Specialist Documents - CMA Case Format*
+
+![A Manual on gov.uk](docs/screen-shot-manual.png)
+*A Manual on gov.uk*
+
+![A Specialist Document on gov.uk](docs/screen-shot-specialist-document-cma-case.png)
+*A Specialist Document on gov.uk*
+
+## Live examples
+
+- [AAIB Reports](https://www.gov.uk/aaib-reports)
+- [CMA Cases](https://www.gov.uk/cma-cases)
+- [Countryside Stewardship Grants](https://www.gov.uk/countryside-stewardship-grants)
+- [DFID Research Outputs](https://www.gov.uk/dfid-research-outputs)
+- [Drug Safety Update](https://www.gov.uk/drug-safety-update)
+- [ESI Funds](https://www.gov.uk/european-structural-investment-funds)
+- [International Development Funds](https://www.gov.uk/international-development-funding)
+- [Medical Safety Alerts](https://www.gov.uk/drug-device-alerts)
+- [MAIB Reports](https://www.gov.uk/maib-reports)
+- [RAIB Reports](https://www.gov.uk/raib-reports)
+- [Vehicle Recalls and Faults Alerts](https://www.gov.uk/vehicle-recalls-faults)
+- Manuals: There is no public index page for Manuals. They can be found at `gov.uk/guidance/:manual-slug`.
+
 ## Nomenclature
 
-* Specialist Documents: Documents with metadata which are published to Finders
-* Schema: JSON file defining slug, document noun and name of Specialist Document document_types. Also has select facets and their possible values for each document_type which are displayed by the `_form.html.erb`.
-* Manual: Grouped Documents published as a number of sections inside a parent document
 
-## Current formats
+- **Format**: Category of a Document. Format names are listed in the `Live Examples` section above and include `MAIB Reports` and `CMA Cases`.
+- **Document**: Specialist Documents are created by Government editors and can be published to gov.uk. Documents differ from each other depending on their format. These differences are largely determined by what is contained in the [schema](https://github.com/alphagov/specialist-publisher-rebuild/blob/add-dfid-review-status/lib/documents/schemas/aaib_reports.json) of a format.
+- **Schema**: JSON files defining attributes for each format, including `base_path`, `document_noun` and `document_type`. It also contains the facets and their possible values for each `document_type` which are displayed by `_form.html.erb`.
+- **Manual**: Grouped Documents published as a number of sections inside a parent document.
```

Comment:
I've often heard people use the term "finder" to refer to a format, maybe we could mention something about that here?